src/Appwrite/Platform/Workers/Certificates.php

```diff
@@ -198,9 +201,11 @@ private function execute(
             $success = true;
         } catch (Throwable $e) {
             $logs = $e->getMessage();
+            $currentLogs = $certificate->getAttribute('logs', '');
+            $date = \date('H:i:s');
+            $errorMessage = "\033[90m[{$date}] \033[31mCertificate generation failed: \033[0m\n";
 
-            // Set exception as log in certificate document
-            $certificate->setAttribute('logs', \mb_strcut($logs, 0, 1000000));// Limit to 1MB
+            $certificate->setAttribute('logs', $currentLogs . $errorMessage . \mb_strcut($logs, 0, 1000000));// Limit to 1MB
```

Comment:
```suggestion
            $certificate->setAttribute('logs', $currentLogs . $errorMessage . \mb_strcut($logs, 0, 500000));// Limit to 500kb
```

1 MB is attribute max, and we expect a lot of other logs to be there, so let's limit this to 50% (500kb). I would anyway never expect more than a few sentences in this log.